src/components/style/navBarStyle.css

```diff
@@ -1,3 +1,5 @@
+/* Curious - why is this a .css, while most are .css.js ? */
+
```

Comment:
this one is css because we needed to override a bunch of classes that are not able to be inline styled from the react bootstrap component level.  We were able to successfully style the navbar this way.  We will try to get everything into this format.